src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/nomis/NomisHistory.kt

```diff
@@ -17,7 +17,7 @@ data class NomisHistory(
   val questions: List<NomisHistoryQuestion>,
   @Schema(description = "When the questionnaire was changed")
   @Deprecated("Use incidentChangeDateTime instead")
-  val incidentChangeDate: LocalDate,
+  val incidentChangeDate: LocalDate? = null,
```

Comment:
Is the sync service always passing the `incidentChangeDateTime` field now? I do wonder if it's worth removing this deprecated field altogether instead of just making it nullable?

But I may miss something, sorry.

Reply:
ah true - we need to sync pushing to prod, but yes, you're right